### ISSUE TITLE
Continue to grow the library compatibility test-kit

### ISSUE THREAD

**Issue by bishabosha** [scalacenter/scala#13]
**[open]**

Find more libraries/examples to add to https://github.com/scalacenter/tasty-reader-compat-testkit

We also need a better way to automate the build of the project, and potentially trigger CI whenever we push to `scalacenter/scala:tasty_reader`

**Comment by bishabosha** [comment]
comment from @nafg:
>regarding testing the tasty reader compiler, what if you could get sbt to take any project that has Compile and Test configs, and compile the project with dotty and run the tests using a tasty-reading scala 2?

**Comment by ashwinbhaskar** [comment]
@bishabosha is this concerned with adding libraries written in dotty to https://github.com/scalacenter/tasty-reader-compat-testkit ?

**Comment by bishabosha** [comment]
@ashwinbhaskar yes, currently thats the method I'm using to test real-world compatibility. I would say that test cases that do not depend on a library can be added directly in this codebase.

**Comment by ashwinbhaskar** [comment]
@bishabosha is there an issue for - https://github.com/scalacenter/tasty-reader-compat-testkit/issues/2#issuecomment-626223765 ? I glanced through the issues to see if there is one. I did not find one (maybe I missed it). Just confirming

**Comment by bishabosha** [comment]
There is one now, thanks
